.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Please see https://github.com/seqan/seqan3/blob/master/CONTRIBUTING.md for a general overview.
+
+Please allow edits from maintainers:
+https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests
+
+When the PR is opened, clang-format will be run on the PR, and any formatting changes will be pushed to your fork.
+This means that you will have to `git pull` the changes before pushing new commits.
+After that, the actual CI will start.
+
+If you want to incorporate the format changes, we recommend using git absorb: https://github.com/tummychow/git-absorb
+
+git absorb distributes staged changes to existing commits.
+The verbose workflow could look like this:
+```
+git pull # pull formatting changes
+git reset --soft HEAD~1 # reset index to before the formatting changes, edited files are staged
+git absorb # run git absorb
+git log # check the auto-generated fixup commits
+git rebase -i --autosquash master # replace master with the branch you branched from if it was not master
+git push -f # force push updates
+```
+
+A shorter, but potentially with unintended squashes, version is:
+```
+git pull # pull formatting changes
+git reset --soft HEAD~1 # reset index to before the formatting changes, edited files are staged
+git absorb --and-rebase master # run git absorb and rebase on master, replace master with the branch you branched from
+git push -f # force push updates
+```
```

Comment:
Hm I think I am against this because it might be complicated for people not familiar with git. The easiest thing would be that we squash merge the contribution. And if it is a more complex contribution requiring more commits, we should maybe rebase/absorb the stuff for them.

This is good for a wiki entry though for our team.